app/src/main/java/com/example/bankoflunar/dataStore/room/deposit_db/DepositDetailsRoomDatabase.kt

```diff
@@ -0,0 +1,39 @@
+package com.example.bankoflunar.dataStore.room.deposit_db
+
+import android.app.Application
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+
+@Database(entities = [DepositDetails::class], version = 1, exportSchema = false)
+
+abstract class DepositDetailsRoomDatabase: RoomDatabase() {
```

Comment:
The name of the database shouldn't be based on the name of one of it's tables. I think a better name would be `BankOfLunarRoomDatabase` or `AppDatabase`